pubsub/topic_test.go

```diff
@@ -122,6 +122,9 @@ func TestTopic_IngestionKinesis(t *testing.T) {
 				AWSRoleARN:        "fake-aws-role-arn",
 				GCPServiceAccount: "fake-gcp-sa",
 			},
+			PlatformLogsSettings: &PlatformLogsSettings{
```

Comment:
nit: Users are allowed to set Platform Logs settings on Kinesis ingestion topics but we don't actually do any platform logging for Kinesis ingestion. Fine with keeping it or removing it.